bundles/org.openhab.binding.folderwatcher/README.md

```diff
@@ -78,3 +93,16 @@ then
 
 end
 ```
+
+S3 bucket example:
+
+```java
+rule "New S3 file"
+when 
+    Channel 'folderwatcher:s3bucket:XXXXX:news3file' triggered
```

Comment:
That would be a next pull request. While developing this enchantment I noticed that for FTP and local folder "newfile" channel is used instead of corresponding "newftpfile" and "newlocalfile"

Reply:
That would also be an improvement, but i menat just to align the docs between the Thing configuration and the example:
Change:
`Channel 'folderwatcher:s3bucket:XXXXX:news3file' triggered`
to:
`Channel 'folderwatcher:s3bucket:myS3bucket:news3file' triggered`

Reply:
Got it, also as per other comment using "newfile" channel instead